.travis.yml

```diff
@@ -2,6 +2,7 @@ language: node_js
 node_js:
   - 10
 before_install:
+  - npm install -g npm@6.13
```

Comment:
I suppose this change is related to my previous comment!

Reply:
Yup